src/components/aeAddress/aeAddress.vue

```diff
@@ -1,9 +1,26 @@
 <template>
-  <div :class='cssClass'>
-    <ae-identity-avatar v-if='showAvatar' :address='address' />
-    <span class='address'>{{displayAddress}}</span>
+  <div :class='cssClass' class="aeAddressWrapper">
```

Comment:
I'm really not a fan of using :class and class on the same element. I prefer something like

```
:class="['aeAddressWrapper', cssClass]"
```

Reply:
No, not that I know. Maybe Vue is slightly more performant one variation Idk. My reason is that I like to have it in one place.